src/scripts/templates/targeted/Find HTML comments.js

```diff
@@ -3,25 +3,25 @@
 function invokeWith(msg) {
 	// Debugging can be done using println like this
 	print('Finding comments under ' + msg.getRequestHeader().getURI().toString()); 
-	prefix = msg.getRequestHeader().getURI().toString();
+	var prefix = msg.getRequestHeader().getURI().toString();
 
-	extHist = org.parosproxy.paros.control.Control.getSingleton().
+	var extHist = org.parosproxy.paros.control.Control.getSingleton().
 	    getExtensionLoader().getExtension(
 	        org.parosproxy.paros.extension.history.ExtensionHistory.NAME) 
 	if (extHist != null) {
-	    i=1
-	    while (hr = extHist.getHistoryReference(i), hr) {
+	    var i=1
+	    while (var hr = extHist.getHistoryReference(i), hr) {
```

Comment:
This breaks the script: `ParserException: <eval>:13:12 Expected an operand but found var`.

Reply:
Ugh, my bad. Fixed and tested this time ;)